c/include/libsbp/ssr.h

```diff
@@ -126,6 +129,9 @@ typedef struct SBP_ATTR_PACKED {
 typedef struct SBP_ATTR_PACKED {
   sv_id_t sv_id;       /**< space vehicle identifier */
   s16 residual;    /**< STEC residual [0.04 TECU] */
+  u8 stddev;      /**< stddev [modified DF389 scale; class is upper 3 bits, value is lower 5
+stddev <= (3^class * (1 + value/16) - 1) * 10 TECU
```

Comment:
Why `* 10`? 1 TECU is ~16cm on L1, so we probably want resolution down to about 100th of a TECU. I think this means we should divide by 100, right?

Reply:
so this is the resolution the formula gives you (in TECU):
```
0
0.0625
0.125
0.1875
0.25
0.3125
0.375
0.4375
0.5
...
```
So divide that by 10, and you should have 1 mm resolution.  Seems like enough, ya?  The tropo is way overkill, but unless we think we need more range, I figured I would stick to mm for the units.

Reply:
Ok, so maybe divide by 10 is more appropriate than divide by 100, but you've got a multiplication in there. If `class` and `value` are the encoded message and `stddev` is the decoded amount, I think we need to divide in order to get more resolution (e.g. 1, 2 -> 0.1, 0.2 with 0.1 resolution; if we multiple, 1, 2 -> 12, 20 with 10 resolution). Isn't that the case?

Reply:
I disagree; let's say we calculate that our stddev is 0.06 TECU.  That's the smallest value the encoding will allow.   So in orion, I multiply by 10 before sending it; now the raw value that gets encoded is 0.6.   When I unpack, I divide by 10 again to get back to TECU.  So what is coming down in the message is TECU * 10, ya?  Or should i call it deci-TECU to eliminate this confusion?

Reply:
All of what you said in that previous comment is correct. Either I've misunderstood the code or the code doesn't correspond, though. Isn't `stddev <= (3^class * (1 + value/16) - 1) * 10 TECU` the code that does the unpacking? I.e. shouldn't it do as you say: `When I unpack, I divide by 10 again to get back to TECU` and not multiply by 10?

Reply:
ah, no - that is just a comment describing what is in that field.  The code that does the unpacking is here:
https://github.com/swift-nav/starling/blob/1bd3efca582e5b15761809e572ea03db5845e3a4/src/pvt_engine/ssr_unpackers.cc#L163

Reply:
alright - going ahead here since I'm out for the rest of the week.  Can always fix the comments later....